libs/utils/utils.js

```diff
@@ -404,6 +404,9 @@ export const loadScript = (url, type) => new Promise((resolve, reject) => {
     if (type) {
       script.setAttribute('type', type);
     }
+    if (mode) {
```

Comment:
I would like us to be more precise with the attributes we're setting here.
```suggestion
    if (['async', 'defer'].includes(mode)) script.setAttribute(mode, true);
```